accounts-db/src/accounts_db.rs

```diff
@@ -654,6 +656,7 @@ pub struct AccountsDbConfig {
     /// The low and high watermark sizes for the read cache, in bytes.
     /// If None, defaults will be used.
     pub read_cache_limit_bytes: Option<(usize, usize)>,
+    pub read_cache_evict_sample_size: Option<usize>,
```

Comment:
```suggestion
    /// The number of elements that will be randomly sampled at eviction time, the oldest of which
    /// will get evicted.
    pub read_cache_evict_sample_size: Option<usize>,
```

Reply:
Done. Not sure why github doesn't show this commend as "outdated"